service/javadsl/kafka/client/src/main/scala/com/lightbend/lagom/internal/javadsl/broker/kafka/JavadslKafkaSubscriber.scala

```diff
@@ -101,17 +104,33 @@ private[lagom] class JavadslKafkaSubscriber[Message](kafkaConfig: KafkaConfig, t
     }
   }
 
-  private def locateService(name: String): Future[Option[URI]] = serviceLocator.locate(name).toScala.map(_.asScala)
+  private def locateService(name: String): Future[Seq[URI]] =
+    serviceLocator.locateAll(name).toScala.map(_.asScala)
 
   override def atLeastOnce(flow: Flow[Message, Done, _]): CompletionStage[Done] = {
-    val streamCompleted = Promise[Done]
-    val consumerProps = KafkaSubscriberActor.props(kafkaConfig, consumerConfig, locateService,
-      topicCall.topicId().value(), flow.asScala, consumerSettings, subscription, streamCompleted)
 
-    val backoffConsumerProps = BackoffSupervisor.propsWithSupervisorStrategy(
-      consumerProps, s"KafkaConsumerActor$consumerId-${topicCall.topicId().value}", consumerConfig.minBackoff,
-      consumerConfig.maxBackoff, consumerConfig.randomBackoffFactor, SupervisorStrategy.stoppingStrategy
-    )
+    val streamCompleted = Promise[Done]
```

Comment:
Same kind of formatting as explained above. There a few more, I won't comment on all of them. :-)